suite/auto-sync/src/autosync/MCUpdater.py

```diff
@@ -146,7 +146,14 @@ def init_tests(self, unified_test_cases: bool):
         log.debug(f"llvm-mc result: {mc_output}")
         text_section = 0  # Counts the .text sections
         asm_pat = f"(?P<asm_text>.+)"
-        enc_pat = r"(\[?(?P<full_enc_string>(?P<enc_bytes>((0x[a-fA-F0-9]{1,2}[, ]{0,2}))+)[^, ]?)\]?)"
+        byte_pat = r"0[bx][a-fA-F0-9]+"
```

Comment:
As it looks like this pattern now also matches binary with the `0b` prefix?
Please check the extracted bytes in `valid_byte_seq()`. Because I remember there are `assembler` tests which put `A` into the bitstring (`0x111101010AAA0111`) to mark where to put address bits.

Also, you need to handle those binary numbers in `MCTest:__str__()`. I think yaml doesn't support mixed hex and binary numbers (if binary at all).

Then please add a unit test for it. I think you can start with `test_mcupdater.py::test_no_symbol_tests()` and edit it to your liking.

Reply:
This part is not important, I can delete this part and submit it again. Because in the end I was not able to implement the symbol injection test.

But this part of the test was too important for xtensa, so I manually replaced the label with the actual address.